fe/fe-core/src/main/java/org/apache/doris/analysis/ShowDataStmt.java

```diff
@@ -100,26 +100,64 @@ public class ShowDataStmt extends ShowStmt implements NotFallbackInParser {
             new ImmutableList.Builder<String>().add("TableName").add("IndexName").add("Size").add("ReplicaCount")
                     .add("RowCount").add("RemoteSize").build();
 
+    private static final ShowResultSetMetaData SHOW_DETAILED_TABLE_DATA_META_DATA =
+            ShowResultSetMetaData.builder()
```

Comment:
columns name and order suggestion:
TableName
ReplicaCount
LocalTotalSize
LocalDataSize
LocalIndexSize
RemoteTotalSize
RemoteDataSize
RemoteIndexSize

Reply:
done